examples/forms.amp.html

```diff
@@ -57,6 +57,34 @@
             color: #dc4e41;
         }
 
+        .verification-form label {
```

Comment:
This is just for the example page we use for development, right? It isn't baked in to `amp-form`. The publisher should be responsible for the spinner UX